pyproject.toml

```diff
@@ -89,6 +89,8 @@ extra-dependencies = [
   "transformers[torch,sentencepiece]==4.47.1", # ExtractiveReader, TransformersSimilarityRanker, LocalWhisperTranscriber, HFGenerators...
   "huggingface_hub>=0.27.0",                   # Hugging Face API Generators and Embedders
   "sentence-transformers>=3.0.0",              # SentenceTransformersTextEmbedder and SentenceTransformersDocumentEmbedder
+  "optimum[onnxruntime]>=1.24.0",              # ONNX CPU runtime for Sentence Transformers components
+  "optimum-intel[openvino]>=1.22.0",           # OpenVINO runtime for Sentence Transformers components (older versions will not work with transformers>=4.7.0)
```

Comment:
since integration tests were transformed into unit tests, these test dependencies can be removed now.
Right, @lbux?